src/klipse/compiler.cljs

```diff
@@ -20,7 +22,34 @@
 
 (def ^{:dynamic true
        :doc "The compiler to use. It could be either :core or :replumb"}
-  *compiler-name* :replumb)
+  *compiler-name* :core)
+
+(def ^{:export true
+       :dynamic true}
+  starttime 0)
+
+(defn ^{:export true}
+  guard []
+  (js/console.log (system-time) starttime (- (system-time) starttime))
+  (when (> (- (system-time) starttime) 1000)
+    (throw (str "Infinite Loop"))))
+
+(def original-emits emits)
+(defn my-emits [& xs]
+  (doseq [x xs]
+    (when (and (string? x) (re-matches #"^(if|continue).*" x))
```

Comment:
how'd you decide on `if` and `continue`? should we also add `recur`?

in general, what is this regex looking for?